### PR TITLE
Allow User To Toggle PR WebHook

### PR DESCRIPTION
**Pull request by yakschuss**
When a user clicks on a repo, it triggers a request to Github which
creates a webhook. If the webhook was already active, it removes the
webhook. Created UI to reflect changes in Webhook Status.

### CODE REVIEW

File: db/migrate/20161101183239_add_active_to_repo.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddActiveToRepo < ActiveRecord::Migration[5.0]
+  def change
+    add_column :repos, :active, :boolean
```
Comment (alexstophel) [inline db/migrate/20161101183239_add_active_to_repo.rb]: You're leaving yourself open for the three-state boolean (`true`, `false` and `nil`). You should add `null: false, default: false` to protect yourself.

File: lib/github_client.rb [hunk 2]
```diff
@@ -24,7 +36,25 @@ def get_user_repos
...
+      active: true,
+    )
+    repo.update_attributes(hook_id: response[:id])
+  end
+
+  def turn_webhook_off(repo)
+    client.remove_hook(repo.full_name, repo.hook_id)
+  end
+
   def client
```
Comment (alexstophel) [inline lib/github_client.rb]: This could just be an `attr_reader` now.

File: app/assets/javascripts/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+$(document).on('turbolinks:load', function() {
```
Comment (alexstophel) [inline app/assets/javascripts/dashboard.js:1]: Blah. Turbolinks is the worst.

File: app/models/repo.rb [hunk 4]
```diff
@@ -3,4 +3,14 @@ class Repo < ApplicationRecord
 
   validates :full_name, presence: true
   validates :name, presence: true
+
+  scope :ordered_by_active, -> { order("active desc") }
+
+  def toggle_active
+    if active?
```
Comment (alexstophel) [inline app/models/repo.rb]: You can replace this entire method with just: `update_attributes(active: !active?)`

File: app/assets/javascripts/dashboard.js (hunk 3 above)
Comment (yakschuss) [reply app/assets/javascripts/dashboard.js:1]: Heh.

File: lib/github_client.rb (hunk 2 above)
Comment (yakschuss) [reply lib/github_client.rb]: Good point! I'll fix that.

**Pull request merged**